src/werkzeug/serving.py

```diff
@@ -307,6 +308,11 @@ def handle(self):
             self.initiate_shutdown()
 
     def initiate_shutdown(self):
+        warnings.warn(
+            "warning",
```

Comment:
Incorrect arguments to `warnings.warn`. Should be `message, type, stacklevel`, see other places in Werkzeug for examples.

Reply:
Hmm can't seem to get it to log properly. I'll keep trying.

Reply:
Oh wait, this is code that likely wouldn't run during tests, so you won't see deprecation warnings unless you're running with `python -Xdev` or otherwise enabling them. In that case I think it's fine to raise a `UserWarning` instead (which is the default if no type is given).